src/components/content-tab-snapshots.tsx

```diff
@@ -182,16 +193,22 @@ function SnapshotRow( {
 				) : (
 					<>
 						<Tooltip
-							disabled={ ! isOffline }
-							icon={ offlineIcon }
-							text={ updateDemoSiteOfflineMessage }
+							disabled={ ! ( isOffline || snapshotCreationBlocked ) }
+							{ ...( isOffline && { icon: offlineIcon } ) }
+							text={ isOffline ? updateDemoSiteOfflineMessage : blockedUserMessage }
```

Comment:
What do you think about using a new constant for the props? Similar to the Add demo site tootltip?
I've created this PR with my suggestion: https://github.com/Automattic/studio/pull/388

Reply:
Yeah, I think it is a good idea. It makes the code more readable. I merged the PR into this PR 👍